### PR TITLE
Add update_rasa_x_version workflow

### PR DESCRIPTION
**Pull request by HotThoughts**
- [X] Get the latest stable `rasa-x` version tag from DockerHub
- [X] Update `rasa-x` version tag in `Chart.yaml` if the current version is not the latest one by creating a new PR
- [ ] Config a git user to make a commit

### CODE REVIEW

File: .github/workflows/update_rasa_x_version.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+      - name: Get latest Rasa X minor 🏷
+        run: |
+          DOCKERHUB_TAGS_URL="https://registry.hub.docker.com/v2/repositories/rasa/rasa-x/tags?page_size=10000"
+          LATEST_RASA_X_MINOR=$(curl -s ${DOCKERHUB_TAGS_URL} | jq -r '.results[].name' | grep -vE 'refs|aws|latest|stable|master|ls|list' | sort -Vr | head -n1 | sed 's/[a-z].*//')
+
+          echo "Latest Rasa X minor: ${LATEST_RASA_X_MINOR}"
+          echo "LATEST_RASA_X_MINOR=${LATEST_RASA_X_MINOR}" >> $GITHUB_ENV
+
+      - name: Compare latest Rasa X minor with rasa-xhelm version ⏯
```
Comment (tczekajlo) [inline .github/workflows/update_rasa_x_version.yml:41]:
```suggestion
      - name: Compare latest Rasa X minor with rasa-x-helm version ⏯
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]

File: .github/workflows/update_rasa_x_version.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        with:
+          # The workflow needs to push to a new branch that changes a workflow file.
+          # To do this, we need the rasabot service account.
+          token: ${{ env.RASABOT_GITHUB_TOKEN }}
+
+
+      - name: Get latest Rasa X minor 🏷
+        run: |
+          DOCKERHUB_TAGS_URL="https://registry.hub.docker.com/v2/repositories/rasa/rasa-x/tags?page_size=10000"
+          LATEST_RASA_X_MINOR=$(curl -s ${DOCKERHUB_TAGS_URL} | jq -r '.results[].name' | grep -vE 'refs|aws|latest|stable|master|ls|list' | sort -Vr | head -n1 | sed 's/[a-z].*//')
```
Comment (tczekajlo) [inline .github/workflows/update_rasa_x_version.yml:36]: We should use only a stable version.
```suggestion
          LATEST_RASA_X_MINOR=$(curl -s ${DOCKERHUB_TAGS_URL} | jq -r '.results[].name' | grep -vE 'refs|aws|latest|stable|master|ls|list' | sort -Vr | head -n1 | sed 's/[a-z].*//')
```
```suggestion
          LATEST_RASA_X_MINOR=$(curl -s ${DOCKERHUB_TAGS_URL} | jq -r '.results[].name' | grep -vE 'refs|aws|latest|stable|master|ls|list' | sort -Vr | grep -E "^[0-9.]+$" | head -n1')
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]

File: .github/workflows/update_rasa_x_version.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        with:
+          python-version: 3.7
+
+      - name: Checkout rasa-x-helm github repository 🕝
+        uses: actions/checkout@v2
+        with:
+          # The workflow needs to push to a new branch that changes a workflow file.
+          # To do this, we need the rasabot service account.
+          token: ${{ env.RASABOT_GITHUB_TOKEN }}
+
```
Comment (tczekajlo) [inline .github/workflows/update_rasa_x_version.yml:31]:
```suggestion
```

File: .github/workflows/update_rasa_x_version.yml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  schedule:
+    # Run once a day at 08:00
+    - cron: '0 8 * * *'
+  workflow_dispatch: {}
+
+env:
+  RASABOT_GITHUB_TOKEN: ${{ secrets.RASABOT_GITHUB_TOKEN }}
+  RASABOT_USERNAME: Roberto
+  RASABOT_EMAIL: 43567378+rasabot@users.noreply.github.com
+
```
Comment (tczekajlo) [inline .github/workflows/update_rasa_x_version.yml:13]:
```suggestion
```

**Review by tczekajlo — APPROVED** [review]
LGTM (I've suggested cosmetic changes)

**Comment by HotThoughts** [comment]
@tczekajlo do you have time to look at [the error](https://github.com/RasaHQ/rasa-x-helm/pull/116/checks?check_run_id=1669134362)?

**Pull request merged**